internal/mesh/internal/controllers/sidecarproxy/status/status.go

```diff
@@ -10,110 +10,76 @@ import (
 )
 
 const (
-	StatusConditionDestinationAccepted = "DestinationAccepted"
-
-	StatusReasonMeshProtocolNotFound = "MeshPortProtocolNotFound"
-	StatusReasonMeshProtocolFound    = "MeshPortProtocolFound"
-
-	StatusReasonMeshProtocolDestinationPort    = "DestinationWithMeshPortProtocol"
-	StatusReasonNonMeshProtocolDestinationPort = "DestinationWithNonMeshPortProtocol"
-
-	StatusReasonDestinationServiceNotFound = "ServiceNotFound"
-	StatusReasonDestinationServiceFound    = "ServiceFound"
-
-	StatusReasonDestinationComputedRoutesNotFound = "ComputedRoutesNotFound"
-	StatusReasonDestinationComputedRoutesFound    = "ComputedRoutesFound"
+	StatusConditionDestinationsAccepted = "DestinationsAccepted"
 
+	StatusReasonMeshProtocolNotFound                  = "MeshPortProtocolNotFound"
+	StatusReasonDestinationPortNotFound               = "DestinationPortNotFound"
+	StatusReasonMeshProtocolDestinationPort           = "DestinationWithMeshPortProtocol"
+	StatusReasonDestinationServiceNotFound            = "ServiceNotFound"
+	StatusReasonDestinationComputedRoutesNotFound     = "ComputedRoutesNotFound"
```

Comment:
These statuses are simplified because status condition type should be treated like a key in a map and so it makes more sense to have a single status condition with that type rather than multiple the way it was before.